kitsune/wiki/views.py

```diff
@@ -1353,13 +1354,20 @@ def show_translations(request, document_slug):
     translated_locales = []
     untranslated_locales = []
 
-    translated_locales.append(document.locale)
-    translated_locales.extend(document.translations.all().values_list(
-        'locale', flat=True))
+    # This will hold locale codes only
+    translated_locales_code = []
+    # Makes sure "English (en-US)" is always on top!
+    translated_locales.append((document.locale, LOCALES[document.locale].native))
+
+    # Gets all the locale code for translated locales
+    # Note: en-US is not a translated locale, it's the default
+    translated_locales_code.extend(document.translations.all().values_list('locale', flat=True))
```

Comment:
This is overly verbose. You can simply say `translated_locale_codes = document.translations.values_list('locale', flat=True)`. This is also less confusing since you aren't alternating between working with `translated_locales` and `translated_locale_codes`.